core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.html

```diff
@@ -2,8 +2,11 @@
 <ng-template #enabledComponent>
     <dotcms-template-builder
         [templateLayout]="pageState.layout"
+        (layoutChange)="onLayoutChange($event)"
         data-testId="new-template-builder"
-    ></dotcms-template-builder>
+    >
+        <dot-global-message toolbar-left></dot-global-message>
```

Comment:
Not sure about this because in some cases the left will have the title.

Reply:
I can change it, any thoughts on where we should show the `saving` message?